libtenzir/include/tenzir/curl.hpp

```diff
@@ -181,8 +181,93 @@ class easy {
+    speed_download = CURLINFO_SPEED_DOWNLOAD,
+    speed_download_t = CURLINFO_SPEED_DOWNLOAD_T,
+    speed_upload = CURLINFO_SPEED_UPLOAD,
+    speed_upload_t = CURLINFO_SPEED_UPLOAD_T,
+    header_size = CURLINFO_HEADER_SIZE,
+    request_size = CURLINFO_REQUEST_SIZE,
+    ssl_verifyresult = CURLINFO_SSL_VERIFYRESULT,
+    filetime = CURLINFO_FILETIME,
+    filetime_t = CURLINFO_FILETIME_T,
+    content_length_download = CURLINFO_CONTENT_LENGTH_DOWNLOAD,
+    content_length_download_t = CURLINFO_CONTENT_LENGTH_DOWNLOAD_T,
+    content_length_upload = CURLINFO_CONTENT_LENGTH_UPLOAD,
+    content_length_upload_t = CURLINFO_CONTENT_LENGTH_UPLOAD_T,
+    starttransfer_time = CURLINFO_STARTTRANSFER_TIME,
+    content_type = CURLINFO_CONTENT_TYPE,
+    redirect_time = CURLINFO_REDIRECT_TIME,
+    redirect_count = CURLINFO_REDIRECT_COUNT,
+    private_ = CURLINFO_PRIVATE,
+    http_connectcode = CURLINFO_HTTP_CONNECTCODE,
+    httpauth_avail = CURLINFO_HTTPAUTH_AVAIL,
+    proxyauth_avail = CURLINFO_PROXYAUTH_AVAIL,
+    os_errno = CURLINFO_OS_ERRNO,
+    num_connects = CURLINFO_NUM_CONNECTS,
+    ssl_engines = CURLINFO_SSL_ENGINES,
+    cookielist = CURLINFO_COOKIELIST,
+    lastsocket = CURLINFO_LASTSOCKET,
+    ftp_entry_path = CURLINFO_FTP_ENTRY_PATH,
+    redirect_url = CURLINFO_REDIRECT_URL,
+    primary_ip = CURLINFO_PRIMARY_IP,
+    appconnect_time = CURLINFO_APPCONNECT_TIME,
+    certinfo = CURLINFO_CERTINFO,
+    condition_unmet = CURLINFO_CONDITION_UNMET,
+    rtsp_session_id = CURLINFO_RTSP_SESSION_ID,
+    rtsp_client_cseq = CURLINFO_RTSP_CLIENT_CSEQ,
+    rtsp_server_cseq = CURLINFO_RTSP_SERVER_CSEQ,
+    rtsp_cseq_recv = CURLINFO_RTSP_CSEQ_RECV,
+    primary_port = CURLINFO_PRIMARY_PORT,
+    local_ip = CURLINFO_LOCAL_IP,
+    local_port = CURLINFO_LOCAL_PORT,
+    tls_session = CURLINFO_TLS_SESSION,
+    activesocket = CURLINFO_ACTIVESOCKET,
+    tls_ssl_ptr = CURLINFO_TLS_SSL_PTR,
+    http_version = CURLINFO_HTTP_VERSION,
+    proxy_ssl_verifyresult = CURLINFO_PROXY_SSL_VERIFYRESULT,
+    protocol = CURLINFO_PROTOCOL,
+    scheme = CURLINFO_SCHEME,
+    total_time_t = CURLINFO_TOTAL_TIME_T,
+    namelookup_time_t = CURLINFO_NAMELOOKUP_TIME_T,
+    connect_time_t = CURLINFO_CONNECT_TIME_T,
+    pretransfer_time_t = CURLINFO_PRETRANSFER_TIME_T,
+    starttransfer_time_t = CURLINFO_STARTTRANSFER_TIME_T,
+    redirect_time_t = CURLINFO_REDIRECT_TIME_T,
+    appconnect_time_t = CURLINFO_APPCONNECT_TIME_T,
+    retry_after = CURLINFO_RETRY_AFTER,
+    effective_method = CURLINFO_EFFECTIVE_METHOD,
+    proxy_error = CURLINFO_PROXY_ERROR,
+    referer = CURLINFO_REFERER,
+    cainfo = CURLINFO_CAINFO,
+    capath = CURLINFO_CAPATH,
+    xfer_id = CURLINFO_XFER_ID,
+    conn_id = CURLINFO_CONN_ID,
+    queue_time_t = CURLINFO_QUEUE_TIME_T,
+    used_proxy = CURLINFO_USED_PROXY,
+    lastone = CURLINFO_LASTONE
+  };
+
+  /// Possible return type data when calling `get_info`.
+  using info_data = std::variant<bool, long, std::string_view>;
```

Comment:
The comment should really explain what it means to get a bool, long, or string view, respectively.